save-common/src/commonMain/kotlin/org/cqfn/save/core/result/DebugInfo.kt

```diff
@@ -10,11 +12,23 @@ import kotlinx.serialization.Serializable
  * @property stdout output of the program under test from OUT stream
  * @property stderr output of the program under test from ERR stream
  * @property durationMillis duration of execution in milliseconds
+ * @property countWarnings number of missing and match warnings
  */
 @Serializable
 data class DebugInfo(
     val execCmd: String?,
     val stdout: String?,
     val stderr: String?,
     val durationMillis: Long?,
+    val countWarnings: CountWarnings? = null,
+)
+
+/**
+ * @property missing number of missing warnings
+ * @property match number of match warnings
+ */
+@Serializable
+data class CountWarnings(
+    val missing: Int? = null,
+    val match: Int? = null,
```

Comment:
I think here these fields can be non-nullable. If this info is not available, `CountWarnings` instance won't be created at all.

Reply:
Fixed